pkg/config/config_template.yaml

```diff
@@ -2070,6 +2070,19 @@ api_key:
 #
 # autoconf_template_dir: /datadog/check_configs
 
+## @param autoconf_config_files_poll - boolean - optional - default: false
+## @env DD_AUTOCONF_CONFIG_FILES_POLL - boolean - optional - default: false
+## Should the we check for new/updated integration configuration files on disk.
+#
+# autoconf_config_files_poll: false
+
+## @param autoconf_config_files_poll_interval - integer - optional - default: 60
+## @env DD_AUTOCONF_CONFIG_FILES_POLL_INTERVAL - integer - optional - default: 60
+## How frequently should the Agent check for new/updated integration configuration files (in seconds).
+## NOTE: Logs config from file do not currently support to be dynamically removed.
```

Comment:
Would it make sense to add this to `autoconf_config_files_poll` as well?

Reply:
Yes